layouts/_default/baseof.html

```diff
@@ -20,7 +20,7 @@
     {{ end }}
 
     {{ range .Site.Params.custom_css }}
-      <link rel="stylesheet" href="{{ relURL ($.Site.BaseURL) }}{{ . }}">
+      <link rel="stylesheet" href="{{ relURL (.) }}">
```

Comment:
For your convenience:
```suggestion
      <link rel="stylesheet" href="{{ relURL . }}">
```

I hope that makes it easier to fix!